client/src/components/game/lab2/components/replay.js

```diff
@@ -0,0 +1,168 @@
+                      <td className="replayBreakdown positiveData">
+                        {this.props.score}
+                      </td>
+                    ) : (
+                      <td className="replayBreakdown negativeData">
+                        {this.props.score}
+                      </td>
+                    )}
+                  </tr>
+                  <tr>
+                    <td className="replayBreakdown">
+                      Correctly clicked circles
+                    </td>
+                    {this.props.rightClick - this.props.wrongNoClick > 0 ? (
+                      <td className="replayBreakdown positiveData">
+                        {this.props.rightClick}
+                      </td>
+                    ) : (
+                      <td className="replayBreakdown negativeData">
+                        {this.props.rightClick}
+                      </td>
+                    )}
+                  </tr>
+                  <tr>
+                    <td className="replayBreakdown">
+                      Correctly not clicked circles
+                    </td>
+                    {this.props.rightNoClick > 0 ? (
+                      <td className="replayBreakdown positiveData">
+                        {this.props.rightNoClick}
+                      </td>
+                    ) : (
+                      <td className="replayBreakdown negativeData">
+                        {this.props.rightNoClick}
+                      </td>
+                    )}
+                  </tr>
+                  <tr>
+                    <td className="replayBreakdown">
+                      Incorrectly clicked circles
+                    </td>
+                    {this.props.wrongClick === 0 ? (
+                      <td className="replayBreakdown positiveData">
+                        {this.props.wrongClick}
+                      </td>
+                    ) : (
+                      <td className="replayBreakdown negativeData">
+                        {this.props.wrongClick}
+                      </td>
+                    )}
+                  </tr>
+                  <tr>
+                    <td className="replayBreakdown">Missed clicks</td>
+                    {this.props.wrongNoClick === 0 ? (
+                      <td className="replayBreakdown positiveData">
+                        {this.props.wrongNoClick}
+                      </td>
+                    ) : (
+                      <td className="replayBreakdown negativeData">
+                        {this.props.wrongNoClick}
+                      </td>
+                    )}
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+          <div className="center" style={{ marginLeft: "-25px" }}>
+            {this.props.gamesPlayed === 0 ? (
+              <Button
+                clickMethod={clickFirst}
+                message={"Continue"}
```

Comment:
The margin around this button is really small - maybe add bottom margin so the button isn't touching the bottom of the game area
<img width="1136" alt="Screen Shot 2021-03-08 at 5 08 51 PM" src="https://user-images.githubusercontent.com/54957497/110388448-1fc9be00-8031-11eb-8642-ccc48a5c1dfc.png">